mathics/doc/latex_doc.py

```diff
@@ -594,79 +596,15 @@ def latex(self, doc_data: dict) -> str:
         return text
 
 
-class LaTeXDocumentation(Documentation):
```

Comment:
This class was never used. The class that we use along the code is `LaTeXMathicsDocumentation`.